src/widgets/dragAndDrop.ts

```diff
@@ -222,11 +241,11 @@ function uploadFiles (fetcher, files, fileBase, imageBase, successHandler) {
             contentType: contentType
           })
           .then(
-            _response => {
+            (_response) => {
               console.log(' Upload: put OK: ' + destURI)
               successHandler(theFile, destURI)
             },
-            error => {
+            (error) => {
```

Comment:
this is the file I was changing for this PR. I ended up using any for the HTML elements that are trying to access the savedStyle property which does not exist on an HTMLElement Type.  Also used type 'any' for Files, otherwise it complained about the lastModifiedDate property.